Extensions/NotificationService/NotificationService.swift

```diff
@@ -177,49 +155,9 @@ extension SyncDataDisplay {
 
             notificationContent.userInfo["sentTabs"] = [tab] as NSArray
 
-            // Add tab to the queue.
-            let item = ShareItem(url: urlString, title: title)
-            _ = tabQueue?.addToQueue(item).value // Force synchronous.
-
             presentNotification(title: .SentTab_TabArrivingNotification_NoDevice_title, body: url.absoluteDisplayExternalString)
         }
     }
-}
-
-extension SyncDataDisplay {
-    func presentSentTabsNotification(_ tabs: [NSDictionary]) {
-        let title: String
-        let body: String
-
-        if tabs.isEmpty {
-            title = .SentTab_NoTabArrivingNotification_title
-            #if MOZ_CHANNEL_BETA || DEBUG
-                body = "DEBUG: Sent Tabs with no tab"
-            #else
-                body = .SentTab_NoTabArrivingNotification_body
-            #endif
-        } else {
-            let deviceNames = Set(tabs.compactMap { $0["deviceName"] as? String })
-            if let deviceName = deviceNames.first, deviceNames.count == 1 {
-                title = String(format: .SentTab_TabArrivingNotification_WithDevice_title, deviceName)
```

Comment:
This string `SentTab_TabArrivingNotification_WithDevice_title` is not used anymore I think? If so we can remove from the `String.swift` file